README.md

```diff
@@ -379,17 +411,101 @@ Documentation Forthcoming
 
 ### Modules
 
-#### `loadModuleData`
+#### Loading Modules
 
-Documentation Forthcoming
+[Holocron Modules](#modules) are loaded and composed on 1) the Server when serving the HTML document and 2) in the Browser [Single Page App](https://developer.mozilla.org/en-US/docs/Glossary/SPA) mode via AJAX. We may load Holocron Modules when 1) a path is matched to the URL, or 2) dispatching a Redux action using an HOC and rendering with a React Component.
```

Comment:
I think we should simplify this sentence a bit as it is a bit confusing to read.  How about something like?

```
Holocron modules are loaded on 1). the server when serving the initial HTML and 2) in the Browser when the client JavaScript takes over and the [single page app](...) is initialized.

Holocron Modules are loaded either when a path is matched to a URL using the [child routes configuration](...), or when a React component uses [the Holocron API](...) to load and render a module.
```